package.json

```diff
@@ -27,9 +27,9 @@
     "@cypress/webpack-preprocessor": "^5.15.5",
     "@openshift-console/dynamic-plugin-sdk": "1.4.0",
     "@openshift-console/dynamic-plugin-sdk-webpack": "1.1.1",
-    "@patternfly/react-core": "^5.1.1",
-    "@patternfly/react-icons": "^5.1.1",
-    "@patternfly/react-styles": "^5.1.1",
+    "@patternfly/react-core": "^6.1.1-prerelease.1",
```

Comment:
Should be changed to same versions as console as part of https://issues.redhat.com/browse/CONSOLE-4464